schunk_gripper_interfaces/srv/Scan.srv

```diff
@@ -0,0 +1,8 @@
+# Request
+int32 num_devices  # Number of devices to scan for
```

Comment:
We need to think here about the default `num_devices = 0` that users will call by issuing the service without arguments.